directives/annotated.py

```diff
@@ -357,6 +365,22 @@ def annotate(html, section_name, annotations):
     content = u''.join(result)
     return u''.join([parts[0], parts[1], content, parts[3], parts[4]])
 
+def add_assets(app):
+    app.add_stylesheet(CSS_FILE)
+    app.add_javascript(JS_FILE)
+
+def copy_assets(app, exception):
```

Comment:
Is it really necessary to manually copy the static JS/CSS files like this?

Can we use the static directory of the a-plus-rst-tools HTML theme for the CSS and JS files?
https://github.com/Aalto-LeTech/a-plus-rst-tools/tree/master/theme/aplus/static
Is JS added with `app.add_javascript()` available in the Sphinx HTML template in the `script_files` template variable?
https://github.com/Aalto-LeTech/a-plus-rst-tools/blob/d2dd81fa98a5e7098180de3cc7f8e0c6f134e6df/theme/aplus/layout.html#L55

https://www.sphinx-doc.org/en/master/theming.html#creating-themes
https://www.sphinx-doc.org/en/master/templating.html#script_files

Reply:
If you use the static directory of the a-plus-rst-tools HTML theme, I think that the course repos do not need to manually add the JS/CSS files in any course-specific `_templates/layout.html` template.

Reply:
Files from the theme static directory are always copied to the build output static directory. On the contrary, using custom code for copying the JS and CSS files to the build static directory enables more control: the JS and CSS do not need to be always copied. We don't need to copy the annotated JS and CSS files if the course does not use the annotated directive at all or if the course uses its own versions of the files in its `_static` directory.